src/search-backend/federated-search-backend.ts

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import type { SearchParams } from '../search-params';
+import type { Result } from '@internetarchive/result-type';
+import type { SearchServiceError } from '../search-service-error';
+import { SearchParamURLGenerator } from '../search-param-url-generator';
+import { BaseSearchBackend } from './base-search-backend';
+import type { SearchBackendOptionsInterface } from './search-backend-options';
+
+/**
+ * The FederatedSearchBackend performs a `window.fetch` request to archive.org
+ */
+export class FederatedSearchBackend extends BaseSearchBackend {
+  private servicePath: string;
+
+  constructor(options?: SearchBackendOptionsInterface) {
+    super(options);
+    this.servicePath =
+      options?.servicePath ?? '/services/search/beta/page_production';
```

Comment:
NOTE: Sigh. I wish we would stop putting "beta/" in URL's. They never go away. :)